sechub-pds-solutions/scancode/10-create-image.sh

```diff
@@ -16,7 +16,7 @@ for <docker registry> with tag <version tag>.
 Required: <base image> ; for example ghcr.io/mercedes-benz/sechub/pds-base:v0.32.1
 
 Additionally these environment variables can be defined:
-- SCANCODE_VERSION - Scancode version to use. E.g. 31.2.1
+- SCANCODE_VERSION - Scancode version to use. E.g. 31.2.4
```

Comment:
Is `SCANCODE_VERSION` mandatory or optional?
I think it's mandatory and should be checked in this script.

Please also document here `SPDX_TOOL_VERSION` and `SPDX_TOOL_CHECKSUM`
Where can I find out the version and the checksum and how is the expected formatting?